test/itemlist_formaction.cpp

```diff
@@ -206,7 +202,7 @@ TEST_CASE("process_op(OP_OPENALLUNREADINBROWSER)", "[itemlist_formaction]"){
 		std::ifstream browserFileStream (browserfile.getPath());
 		openedItemsCount = 0;
 		if (browserFileStream.is_open()) {
-			while ( std::getline (browserFileStream,line) ) {
+			while ( std::getline (browserFileStream,line)) {
```

Comment:
Hmm, you removed the space before the closing bracket but left one before the open one. This is just arbitrary, please don't do that.